draft-ietf-quic-transport.md

```diff
@@ -3742,10 +3742,11 @@ ack-eliciting packet.
 
 An endpoint MUST NOT send a non-ack-eliciting packet in response to a
 non-ack-eliciting packet, even if there are packet gaps that precede the
-received packet. This avoids an infinite feedback loop of acknowledgments,
-which could prevent the connection from ever becoming idle.  Non-ack-eliciting
-packets are eventually acknowledged when the endpoint sends an ACK frame in
-response to other events.
+received packet.  Additionally, a receiver MUST NOT send an ack-eliciting
+frame in all packets that would otherwise be non-ack-eliciting. This avoids
+an infinite feedback loop of acknowledgments, which could prevent the
+connection from ever becoming idle.  Non-ack-eliciting packets are eventually
```

Comment:
```suggestion
an infinite feedback loop of acknowledgments.  Non-ack-eliciting packets are eventually
```

I think that the connection between infinite loop and never being idle is clear enough without saying it.